Module.yaml

```diff
@@ -0,0 +1,85 @@
+version: 1.0.0
+name: Melita IO for devices controller
```

Comment:
The name is too long for showing in UI drag and drop. Maybe just change to `Melita IO Controller` or `Melita IO Egress`

Reply:
If you change the name then you need to change all `melita-io` to this new name in code.

Reply:
changed to melita-io-egress